### DISCUSSION TITLE
Name of event in body tag brakes css of whole page

### DISCUSSION THREAD

**Discussion by TomRichtr** [gtm-support/vue-gtm#153] [General]
Every $gtm.trackEvent is pushing string with name of the event into <body></body>. Is it a problem of wrong implementation? Or vue-gtm itself. Can it be avoided?
![Snímek obrazovky pořízený 2022-03-04 09-49-40](https://user-images.githubusercontent.com/65905478/156731780-934621f3-b8c3-4157-b4bd-4849c56f28f3.png)

**Comment by Shinigami92** [comment]
This is what it does: https://github.com/gtm-support/core/blob/c4bbbe51783f33aea3ce187f4463533a5353b315/src/gtm-support.ts#L207

I don't think this is an issue of gtm
